packages/db2/src/_internal/data-types-overrides.ts

```diff
@@ -134,7 +134,7 @@ export class DATE extends BaseTypes.DATE {
   }
 
   toBindableValue(date: AcceptedDate) {
-    date = dayjs(date).utc(false);
+    date = dayjs(date);
```

Comment:
I'm not sure about this change. As far as I can see the default is local time, so this is just an additional thing.
It was causing build issues because we haven't enabled the UTC plugin in this package. Do we need/want to do that? Or do we only want to do that in core? See https://github.com/sequelize/sequelize/blob/main/packages/core/src/utils/dayjs.ts for what we do in core

Reply:
Nevermind, we have failing integration tests so I'll fix this up